src/grdeviceside.cpp

```diff
@@ -198,6 +206,13 @@ namespace rhost {
                 _has_pending_render(false) {
             }
 
+            plot::plot(pDevDesc dd, plot* source_plot) :
```

Comment:
It looks like it's could be unified with the ctor above, with `source_plot` defaulting to `nullptr`.